GVRf/Framework/src/org/gearvrf/scene_objects/GVRCameraSceneObject.java

```diff
@@ -54,8 +64,8 @@
      *            should be sure to call it before you call
      *            {@link Camera#startPreview()}.
      */
-    public GVRCameraSceneObject(GVRContext gvrContext, GVRMesh mesh,
-            Camera camera) {
+    public GVRCameraSceneObject(GVRContext gvrContext, Activity activity,
```

Comment:
sorry if i missed this before:  you can retrieve the activity from the GVRContext.  gvrContext.getActivity();  So there's no need to change the arguments.